cf-java-logging-support-opentelemetry-agent-extension/src/test/java/com/sap/hcf/cf/logging/opentelemetry/agent/ext/binding/CloudLoggingBindingPropertiesSupplierTest.java

```diff
@@ -85,37 +85,53 @@ public void prefersUserProvidedOverManaged() throws Exception {
+            remove("ingest-otlp-endpoint");
+        }};
+        when(servicesProvider.get()).thenReturn(Stream.of(createCfService(BINDING, credentials)));
+        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(servicesProvider);
+
         Map<String, String> properties = propertiesSupplier.get();
+
         assertTrue(properties.isEmpty());
     }
 
     @Test
     public void emptyWithoutClientCert() {
-        String markedService = USER_PROVIDED_VALID.replace("test-client-cert", "");
-        CfEnv cfEnv = new CfEnv("", "{\"user-provided\":[" + markedService + "]}");
-        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(cfEnv);
+        HashMap<String, Object> credentials = new HashMap<String, Object>(CREDENTIALS) {{
+            remove("ingest-otlp-cert");
+        }};
+        when(servicesProvider.get()).thenReturn(Stream.of(createCfService(BINDING, credentials)));
+        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(servicesProvider);
+
         Map<String, String> properties = propertiesSupplier.get();
+
         assertTrue(properties.isEmpty());
     }
 
     @Test
     public void emptyWithoutClientKey() {
-        String markedService = USER_PROVIDED_VALID.replace("test-client-key", "");
-        CfEnv cfEnv = new CfEnv("", "{\"user-provided\":[" + markedService + "]}");
-        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(cfEnv);
+        HashMap<String, Object> credentials = new HashMap<String, Object>(CREDENTIALS) {{
+            remove("ingest-otlp-key");
+        }};
+        when(servicesProvider.get()).thenReturn(Stream.of(createCfService(BINDING, credentials)));
+        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(servicesProvider);
+
         Map<String, String> properties = propertiesSupplier.get();
+
         assertTrue(properties.isEmpty());
     }
 
     @Test
     public void emptyWithoutServerCert() {
-        String markedService = USER_PROVIDED_VALID.replace("test-server-cert", "");
-        CfEnv cfEnv = new CfEnv("", "{\"user-provided\":[" + markedService + "]}");
-        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(cfEnv);
+        HashMap<String, Object> credentials = new HashMap<String, Object>(CREDENTIALS) {{
+            remove("server-ca");
+        }};
+        when(servicesProvider.get()).thenReturn(Stream.of(createCfService(BINDING, credentials)));
+        CloudLoggingBindingPropertiesSupplier propertiesSupplier = new CloudLoggingBindingPropertiesSupplier(servicesProvider);
+
         Map<String, String> properties = propertiesSupplier.get();
+
         assertTrue(properties.isEmpty());
     }
 }
```

Comment:
```suggestion
}

```

Missing new line :-)